### PR TITLE
Update whois.profile

### PR DESCRIPTION
**Pull request by aminvakil**
More details in https://github.com/netblue30/firejail/issues/3153

### CODE REVIEW

File: etc/whois.profile [hunk 1]
```diff
@@ -45,8 +45,8 @@ private
 private-bin bash,sh,whois
 private-cache
 private-dev
-private-etc alternatives,hosts,jwhois.conf,services,whois.conf
-private-lib
+private-etc alternatives,hosts,jwhois.conf,services,whois.conf,resolv.conf
```
Comment (rusty-snake) [inline etc/whois.profile]:
```suggestion
private-etc alternatives,hosts,jwhois.conf,resolv.conf,services,whois.conf
```

Comment (aminvakil) [reply etc/whois.profile]: What's the difference?

**Comment by glitsj16** [comment]
I **wouldn't merge this yet**. IMHO the OP might be using firecfg, in which case the strace examples would need to refer to '/usr/bin/whois' instead of 'whois'. I'm asking some more info on the situation in #3153.

File: etc/whois.profile (hunk 1 above)
Comment (glitsj16) [inline etc/whois.profile]: I'm trying to reproduce this. Please see #3153 for details.

File: etc/whois.profile [hunk 2]
```diff
@@ -45,8 +45,8 @@ private
...
+private-lib gconv
```
Comment (glitsj16) [inline etc/whois.profile]: I'm trying to reproduce this. Plaese see #3153 for details.

File: etc/whois.profile (hunk 1 above)
Comment (Vincent43) [reply etc/whois.profile]: alphabetic sorting

**Comment by glitsj16** [comment]
Can someone confirm whois is working as expected as-is, so without the changes proposed in this PR? It is for me on both Arch and Ubuntu, but after prolonged discussion in #3153 it would be nice to make sure before closing this due to 'not a bug'.

File: etc/whois.profile (hunk 2 above)
Comment (netblue30) [reply etc/whois.profile]: I get it on Debian stable:

$ firejail whois google.com
getaddrinfo(whois.verisign-grs.com): Name or service not known

**Comment by netblue30** [comment]
OK, I'm getting lost in this resolve-conversation thing github put in. So, I would merge @aminvakil, and change:

private-etc alternatives,hosts,jwhois.conf,services,whois.conf,resolv.conf
private-lib gconv

This is working fine on Debian stable.

**Pull request merged**